skins/cat17/src/sass/components/_forms.scss

```diff
@@ -183,6 +183,9 @@ label, input#amount-typed, .wrap-amount-typed:after{
 		.error-text {
 			display: block;
 		}
+		#amount-typed {
```

Comment:
We should avoid using ids for CSS.
See 

- https://stackoverflow.com/questions/8084555/why-selecting-by-id-is-not-recommended-in-css
- http://oli.jp/2011/ids/